docs/usage/self-hosted-configuration.md

```diff
@@ -150,46 +150,21 @@ If the "development branch" is configured but the branch itself does not exist (
 ## binarySource
 
 Renovate often needs to use third-party binaries in its PRs, like `npm` to update `package-lock.json` or `go` to update `go.sum`.
-By default, Renovate uses a child process to run such tools, so they must be:
+By default, Renovate uses a child process to run such binaries, so they must be:
```

Comment:
We need to decide if we want to use `binaries` or `tools` in the docs. Right now we use `tools` and `binaries`, which is a bit confusing. 😄